### PR TITLE
1.1.17 release notes and final EOL announcement

### PR DESCRIPTION
**Pull request by duderino**
1.1.17 will likely be ready to publish on Friday October 18th.  I'll add cherry pick labels then so this doesn't accidentally go out early.

### CODE REVIEW

**Comment by duderino** [comment]
lint failure was a timeout fetching an external URL

File: content/en/news/2019/announcing-1.1-eol-final/index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Support for Istio 1.1 has ended
+description: Istio 1.1 end of life announcement.
+publishdate: 2019-08-18
```
Comment (linsun) [inline content/en/news/2019/announcing-1.1-eol-final/index.md]: needs to be updated

File: content/en/news/2019/announcing-1.1-eol-final/index.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+attribution: The Istio Team
+aliases:
+    - /blog/2019/announcing-1.1-eol-final
```
Comment (geeknoid) [inline content/en/news/2019/announcing-1.1-eol-final/index.md]: No aliases for new files, they're only used when moving old files.

File: content/en/news/2019/announcing-1.1.17/index.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+title: Announcing Istio 1.1.17
+description: Istio 1.1.17 patch release.
+publishdate: 2019-10-18
+attribution: The Istio Team
+release: 1.1.17
+---
```
Comment (geeknoid) [inline content/en/news/2019/announcing-1.1.17/index.md]: I know I just had you remove an alias in the other file, so sorry for making you add one here.

For the sake of having simpler links to release notes from istio/istio, all our release announcements now need a default alias:

```yaml
aliases:
    - /news/announcing-1.1.7
```
Thanks.

Comment (duderino) [reply content/en/news/2019/announcing-1.1.17/index.md]: done

File: content/en/news/2019/announcing-1.1-eol-final/index.md (hunk 1 above)
Comment (duderino) [reply content/en/news/2019/announcing-1.1-eol-final/index.md]: done

File: content/en/news/2019/announcing-1.1-eol-final/index.md (hunk 2 above)
Comment (duderino) [reply content/en/news/2019/announcing-1.1-eol-final/index.md]: done

**Pull request merged**